src/components/BuildingImage/BuildingImage.vue

```diff
@@ -1,6 +1,16 @@
 <template>
   <div class="building-image">
+    <h3
```

Comment:
If the difference is only in the tag name, I would say that `<component :is="isHeader ? 'h3' : 'h2' ">` would be a more DRY & maintainable version of this code

Reply:
Also is the `if` correct? Is it a lower heading level (`h3`) if it's an `isHeader`? Seems to be different in other places, but I might not be seeing the entire picture